src/components/MonthCard.vue

```diff
@@ -43,6 +44,11 @@ export default {
     getLastDayByIndex(i) {
       return moment(this.lastDayOfMonth, 'DD-MM-YYYY').add(i, 'days').format('D');
     },
+    checkIsWeekEnd(day) {
+      const currentDay = moment(`${day}-${this.month}-${this.currentYear}`, 'DD-MM-YYYY').day();
+      const isWeekEnd = currentDay === 0 || currentDay === 6;
+      return isWeekEnd;
+    },
   },
   computed: {
     displayWeekDays() {
```

Comment:
тут можно написать немного иначе
```suggestion
    displayWeekDays() {
       return this.weekDays.map(day => day.slice(0, 2));
    });
```

Reply:
done